ocaml/src/alba.ml

```diff
@@ -435,23 +435,31 @@ let alba_show_object
        >>= fun (hm,r) ->
        match r with
          | None ->
-            Lwt_io.eprintlf "not found"
+            Lwt.fail_with "not found"
```

Comment:
right. we should reserve the right to do whatever we want to do with the json format